src/posthog-surveys.ts

```diff
@@ -0,0 +1,32 @@
+import { PostHog } from "./posthog-core";
+import { SURVEYS } from "./posthog-persistence";
+import { SurveyCallback } from "types";
+
+
+export class PostHogSurveys {
```

Comment:
I'm assuming the other necessary API targeting options will get added to this class in a follow up?

Reply:
Are you talking about for the linked/targeted feature flags?